float32 sideLength
bool clockwise
